Dockerfile-rust

```diff
@@ -2,8 +2,9 @@ FROM rust:latest
 LABEL maintainer="Fastly OSS <oss@fastly.com>"
 
 ENV RUST_TOOLCHAIN=1.83.0
+ENV WASM_WASI_RELEASE=wasm32-wasip1
```

Comment:
This is a 'compilation target', or just 'target', not a 'release'. I'm not sure there is any value in making this an `ENV` though, since it is only referenced in one place.